mmv1/templates/terraform/constants/cloudquotas_quota_preference.go.erb

```diff
@@ -0,0 +1,3 @@
+func QuotaConfigAnnotationsDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
I would guess that we don't want to always suppress diffing on this field. Was this added to support server-controlled annotations?